content/en/developers/faq/how-to-monitor-logs-with-loggly-live-tail-and-datadog.md

```diff
@@ -5,5 +5,5 @@ kind: faq
 
 See the [blog post written by the Loggly team][1].
 
-[1]: https://www.loggly.com/blog/how-to-monitor-logs-with-loggly-live-tail-and-datadog/
```

Comment:
there are a lot of these - why the extra line?

Reply:
I think the script might add a blank line by default when refactoring the links and doesn't check if there is already a blank line --" I will remove all double blank line in another PR